File: app/src/main/java/br/com/module/situationworld/utils/VersionUtils.java
package br.com.module.situationworld.utils;

import android.os.Build;
import android.transition.Explode;
import android.view.Window;

public class VersionUtils {

    public static void checkVersioAndEnableTransition(Window window) {
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP){
            window.requestFeature(Window.FEATURE_CONTENT_TRANSITIONS);
            window.setExitTransition(new Explode());
        }
    }
}

File: app/src/main/java/br/com/module/situationworld/utils/VersionUtils.kt
package br.com.module.situationworld.utils

import android.os.Build
import android.transition.Explode
import android.view.Window

object VersionUtils {
    fun checkVersioAndEnableTransition(window: Window) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
            window.requestFeature(Window.FEATURE_CONTENT_TRANSITIONS)
            window.exitTransition = Explode()
        }
    }
}